### PR TITLE
Main content in the row

### PR DESCRIPTION
**Pull request by eugenekhristo2**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
 * {
     color: purple;
+    font-weight: bold;
+}
+
+body {
+    display: flex;
+    justify-content: space-between;
```
Comment (eugenekhristo) [inline style.css]: Please make it space-around!

Comment (eugenekhristo) [reply style.css]: Ans also make a single meaningful commit message!

Comment (eugenekhristo2) [reply style.css]: Syka OK!

Comment (eugenekhristo2) [reply style.css]: IDK how to squash commits!

**Pull request merged**